templates/.circleci/config.yml

```diff
@@ -244,8 +244,13 @@ workflows:
       jobs:
         - checkout_code
 
+        - lint:
+            requires:
+              - checkout_code
+
         - unit_test:
             requires:
+              - lint
               - checkout_code
```

Comment:
You don't need to specify `checkout_code` since that's already a dependency of `lint`

Reply:
Ah, thanks!